common-content/en/module/tools/discuss-programming-language-concepts/index.md

```diff
@@ -0,0 +1,68 @@
++++
+title = "Discussion: programming language concepts"
+
+time = 60
+emoji = "🧰"
+objectives = [
+  "Define the term variable.",
+  "Explain the purpose of main memory.",
+  "Describe how a variable relates to a memory location.",
+  "Describe how a string is laid out in memory.",
+  "Explain what happens when you call a function.",
+  "Define the term class.",
+  "Explain the relationship between a class and an instance of a class.",
+  "Explain why computers have secondary storage.",
+]
+[build]
+  list = "local"
+  publishResources = false
+  render = "never"
++++
+
+This sprint you have read about programming language concepts in How Computers Really Work.
+
+Topics to discuss:
+
+<!--
+TODO: Make this block more prescriptive about activities
+-->
```

Comment:
Came here to comment this!

As it stands, I think this really opens the door to lecturing (which I recognise is unlikely to happen in the first cohort, but long term I think we'll have to defend against).